local/bin/sh/run-site.sh

```diff
@@ -37,11 +36,12 @@ if [ ${RUN_SERVER} = true ]; then
 	fi
 
 	# webpack
-	if [ ${RUN_WEBPACK} = true ]; then
-		echo "Checking that node modules are installed and up-to-date."
-    npm --global install yarn && \
-    npm cache clean --force && yarn install --frozen-lockfile
-    echo "Starting webpack and hugo build."
+	# if [ ${RUN_WEBPACK} = true ]; then
+	# 	echo "Checking that node modules are installed and up-to-date."
+  #   npm --global install yarn && \
+  #   npm cache clean --force && yarn install --frozen-lockfile
+  #   echo "Starting webpack and hugo build."
+	
```

Comment:
I think we may still need the `yarn install --frozen-lockfile`, you can test by deleting your node modules dir and running make start

Reply:
@davidejones above comment should apply here as well and resolve this